pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -297,6 +298,8 @@ public BrokerService(PulsarService pulsar) throws Exception {
                 .newSingleThreadScheduledExecutor(new DefaultThreadFactory("consumed-Ledgers-monitor"));
         this.ledgerFullMonitor =
                 Executors.newSingleThreadScheduledExecutor(new DefaultThreadFactory("ledger-full-monitor"));
+        this.deduplicationSnapshotMonitor =
```

Comment:
Can we save creating the pool if the feature is disabled

Reply:
It looks like we are not shutting down the pool, leaking threads

Reply:
Oh Yes , I  always forget :sweat_smile: